### ISSUE TITLE
CURLOPT_FOLLOWLOCATION cannot be activated when in safe_mode or an open_basedir is set

### ISSUE THREAD

**Issue by argl** [podlove/podlove-publisher#725]
_This has been moved from podlove-webplayer, where it was posted in the wrong place._

I ran into this on a shared host where I cannot disable open_basedir. It comes up when saving an Episode. Since I am not really on top of PHP these days, is this setting an ancient relict of a long gone time when "safe_mode" ruled the world?

I guess podlove needs proper redirects working at least when analytics are enabled. Is it necessary to implement redirect following in PHP like some people suggest as a workaround?

Having been pointed to a line in the podlove code where the setting is actually checked before the follow option is set, I discovered this:

Hm, good catch. The value is actually checked. I dug a little deeper and got stumped, ```ini_get()``` does not seem to return the ```open_basedir``` setting. I checked with thefollowing code:

```php
<pre>
<?php
echo 'allow_url_fopen = ' . ini_get('allow_url_fopen') . "\n";
echo 'open_basedir = ' . ini_get('open_basedir') . "\n";
?>
</pre>

<pre>
<?php
  print_r(ini_get_all(null, false));
?>
</pre>
```

The output is:
```
allow_url_fopen = 1
open_basedir = 
Array
(
    [allow_url_fopen] => 1
..loads of settings...
    [open_basedir] => /tmp:/home2/www/webmailredirect
)
```

So, ```ini_get_all()``` shows me the correct value, and ```ini_get()``` does not. But only for the ```open_basedir``` key. I am aware that this does not seem to be a podlove publisher problem, but I do not understand that behaviour.

I worked around the problem for now by disabling all error reporting (error_reporting(0);), so the warning does not make it to the output. Still, I am puzzled.

@chemiker asked to use this code:
Could you instead of echo ini_get('open_basedir') use var_dump() and paste the output here?

which i duly did:

```php
var_dump(ini_get('open_basedir'));
```
result:
```
string(0) ""
```

So, long winding post coming to an end, no real solution yet not really sure if this is podlove-related. I have disabled all php warnings for now and thing "work" now.

**Comment by eteubert** [comment]
I just added a fix for the issue that will be released in 2.1.1 soon.
However, it relies on `ini_get('open_basedir')` reporting a path correctly, so it might not help your specific case.

**Comment by argl** [comment]
Thanks, I dont think that this is problem of podlove poublisher anyway, its probably down to some stupid setup on a shared host.
